src/resources/postcss/base/forms/_toggles.pcss

```diff
@@ -66,4 +67,41 @@
 			margin: 0 0 6px;
 		}
 	}
+
+	body#top & { /* Enfold Compatibility */
+
+		.tribe-common-form-control-toggle {
+
+			input[type="checkbox"] {
+				@mixin form-control-slider-toggle-track;
```

Comment:
same question on whether we need all these mixins / styles or if it's just a certain number of them.

Reply:
only one! Ready

Reply:
Also the `&:after` and the `label` below this.

Reply:
Done @paulmskim